android/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,15 @@
     package="com.ee2_mobile_app">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <!-- Required -->
+    <uses-permission android:name="android.permission.CAMERA" />
+
+    <!-- Include this only if you are planning to use the camera roll -->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+
+    <!-- Include this only if you are planning to use the microphone for video recording -->
+    <uses-permission android:name="android.permission.RECORD_AUDIO"/>
```

Comment:
Remove this

Reply:
ok

Reply:
I mean remove the unused permissions, not remove the comments. Please read the permissions before put all of them to the app.

Reply:
Okay, I have removed RECORD_AUDIO permission.